_posts/2016-12-06-recurse-center-week-10.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Week[9] @ The Recurse Center 🔎"
+comments: true
+categories: tech recurse
+---
+
+### Overall
+
+I'm currently trying to distill through what is essential, and focus on that. Stripping away the excess has left more energy for the important pursuits.
+
+### Emotional 😰
+
+### **Frantic**
+
+Feels have been evolving recently, but at the moment I feel frantic. There's now only **2** weeks left.
+
+I was reflecting on the first half versus the second half. It seems like I got more done in the first half...
```

Comment:
I know exactly how this feels. I certainly felt like I was coasting on the momentum I had in my first half, but then I was describing weeks 7-9 to an alum and to him it sounded like I had accomplished a lot. Try doing the same thing and you might be pleasantly surprised at all you've done :smile:. Remember also that I devoted my entire batch to getting better at Haskell after spending a couple of months studying it on my own so from my perspective you've made a ton of progress in a few weeks.